examples/synthetic_data/syn_text_pii_scanner_example.ipynb

```diff
@@ -0,0 +1,608 @@
+{
```

Comment:
<p>State that the predictions will be appended to the data object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/LeakPro/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>